global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "5.0.102",
+    "version": "6.0.100-preview.2.21155.3",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "5.0.102",
+    "dotnet": "6.0.100-preview.2.21155.3",
     "vs": {
       "version": "16.8"
     },
     "xcopy-msbuild": "16.8.0-preview2.1"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.21161.1",
-    "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.21161.1"
+    "Microsoft.DotNet.Arcade.Sdk": "6.0.0-beta.21167.3",
+    "Microsoft.DotNet.Helix.Sdk": "6.0.0-beta.21167.3"
   }
-}
+}
```

Comment:
📝 Unexpected removal of EOL. If this action was performed by a tool, please also report a bug on that tool.

Reply:
Hmm.. no telling. We could enforce this by updating the .edtitorconfig https://github.com/dotnet/roslyn/blob/main/.editorconfig#L27

Reply:
That would only work on tools that understand **.editorconfig**. Since the default behavior should always be "preserve existing", it should not be a problem to omit it from the configuration.